CONTRIBUTING.md

```diff
@@ -51,6 +51,23 @@ npm link
 
 You can then run the Functions Framework locally using `functions-framework`.
 
+## Publishing (Admins only)
+
+This module is published using Release Please. When you merge a release PR, the npm package will be automatically published.
+
+### Reverting a Publish
+
+If the release process fails, you can revert the publish by running the following (i.e. unpublishing `1.10.0`):
+
+```sh
+# Login to the Wombat Dressing Room. Create a 24 hour token. Close the window.
+npm login --registry https://wombat-dressing-room.appspot.com
+# Unpublish the package (must be done within 24 hours of publishing).
```

Comment:
What do you do if it's been > 24 hours? 🤔

Reply:
Good point. I'll add a bit more detail and a link to the npm docs.